packages/flutter/lib/src/material/progress_indicator.dart

```diff
@@ -574,8 +574,7 @@ class CircularProgressIndicator extends ProgressIndicator {
   /// [CupertinoActivityIndicator] in [TargetPlatform.iOS] & [TargetPlatform.macOS] and [CircularProgressIndicator] in
   /// material theme/non-Apple platforms.
   ///
-  /// The [value], [valueColor], [strokeWidth], [semanticsLabel], and
-  /// [semanticsValue] will be ignored in iOS & macOS.
+  /// [valueColor], [strokeWidth], [semanticsLabel], and [semanticsValue] will be ignored in iOS & macOS.
```

Comment:
`on` instead of `in`?
```suggestion
  /// [valueColor], [strokeWidth], [semanticsLabel], and [semanticsValue] will be ignored on iOS & macOS.
```